src/common/proc_macro/src/lib.rs

```diff
@@ -48,10 +54,172 @@ mod config;
 #[cfg_attr(coverage, no_coverage)]
 #[proc_macro_derive(OverrideConfig, attributes(override_opts))]
 #[proc_macro_error]
-pub fn override_config(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
+pub fn override_config(input: TokenStream) -> TokenStream {
     let input = parse_macro_input!(input);
 
     let gen = config::produce_override_config(input);
 
     gen.into()
 }
+
+#[proc_macro_derive(EstimateSize)]
+pub fn derive_estimate_size(input: TokenStream) -> TokenStream {
```

Comment:
I think `GetSize` crate has implemented similar function? If so, do we really need to maintain one for this?